airbyte-webapp/src/components/connectorBuilder/Builder/Builder.module.scss

```diff
@@ -8,15 +8,18 @@
 }
 
 .sidebar {
-  @include mixins.right-shadow;
+  @include mixins.shadow;
 
-  width: 200px;
+  width: 220px;
```

Comment:
This and the change below are just to bring the builder more in line with the current designs, and to make it look slightly better when part of the main view